plugins/flutter_aepedge/lib/src/aepedge_experienceevent.dart

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/// Experience Event is the event to be sent to Adobe Experience Platform Edge Network.
+class ExperienceEvent {
+  static const String _xdmDataKey = 'xdmData';
+  static const String _dataKey = 'data';
+  static const String _dataIdentifierKey = 'dataIdentifier';
+
+  ExperienceEvent(this.eventData);
+
+  ExperienceEvent.createEvent(final Map<String, dynamic> xdmData,
+      [final Map<String, dynamic>? data, final String? datasetIdentifier]) {
+    final Map<String, dynamic> experienceEventConstructorData = {
+      _xdmDataKey: xdmData,
+      _dataKey: data,
+      _dataIdentifierKey: datasetIdentifier
+    };
+    this.eventData = experienceEventConstructorData;
+  }
+
+  /// The data in this experience event.
+  late Map<String, dynamic> eventData;
```

Comment:
this should be private _eventData

Reply:
Need to call experienceEvent.eventData from sendEvent, can't make this to private
https://github.com/cacheung/aepsdk_flutter/blob/EdgeUpdate/plugins/flutter_aepedge/lib/flutter_aepedge.dart#L33

Reply:
@emdobrin looked at this and we keep it not private right now.